subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -140,19 +140,15 @@ int lll_init(void)
 	event.curr.abort_cb = NULL;
 
 	/* Initialize LF CLK */
-	clk_k32 = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL "_32K");
-	if (!clk_k32) {
+	clk = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL);
+	if (!clk) {
 		return -ENODEV;
 	}
 
-	clock_control_on(clk_k32, NULL);
+	clock_control_on(clk, NRF_CLOCK_SUBSYS_LF);
 
 	/* Initialize HF CLK */
-	lll.clk_hf =
-		device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL "_16M");
-	if (!lll.clk_hf) {
-		return -ENODEV;
-	}
+	lll.clk_hf = clk;
```

Comment:
Would it make sense to rename this structure member too?

Might be out of scope for this PR.

Reply:
done